### ISSUE TITLE
Accessible foreground colour generator

### ISSUE THREAD

**Issue by michael-milette** [ThePacielloGroup/CCAe#257]
Thank you so much for your continued support and generosity in sharing, CCA, a very useful tool indeed.

I often find it time consuming and sometimes even a frustrating experience to try and find WCAG compliant colour combinations that will correct a colour contrast issue.

Once I've selected the current foreground and background colours, and determined that they lack contrast, I wish CCA had a button which would instantly generate a compliant and closely matching foreground colour for the given background colour.

The best solution I have come across, which I have been using since I discovered it, is the following very helpful web-based tool called the Accessible Color Generator (ACG):
https://www.learnui.design/tools/accessible-color-generator.html

**Screenshot:**
![image](https://user-images.githubusercontent.com/3808579/181708957-00140655-7e38-413c-bcb0-5a20a0b3234f.png)

Having something similar integrated into CCA would eliminate the steps of having to copy 2 colours from CCA and pasting them into ACG in order to get its recommendation.

Best regards,

Michael Milette
